docs/ansible/roles/debops.prosody/getting-started.rst

```diff
@@ -0,0 +1,70 @@
+Getting started
+===============
+
+.. contents::
+   :local:
+
+
+Default configuration
+---------------------
+
+The configuration is split into 3 basic parameters,
+this is because of limitation of YAML and easier representation.
+
+- prosody__*_config_global
+- prosody__*_config_components
+- prosody__*_config_virtual_hosts
+
+By default there are two components active :envvar:`prosody__http_upload` :envvar:`prosody__muc`
+Set this varables to false to disable the specific component.
+
+Domains
+~~~~~~~
+
+The default virtual host uses :envvar:`prosody__domain` as domain.
+
+The components uses two subdomains: conference.`prosody__domain and upload.`prosody__domain`
+
+Ports
+~~~~~
+
+By default the ports are:
+- `5222` (c2s)
+- `5269` (s2s)
+- `5280` http
+- `5281` https
+
+
+
+Example inventory
+-----------------
+
+``debops.prosody`` is included by default in the :file:`common.yml` DebOps playbook;
+To enable it, add a host to the group `[debops_service_prosody]`.
```

Comment:
This is false, role is not included in the common playbook. You can replace this text with something similar to [NFS server inventory](https://docs.debops.org/en/latest/ansible/roles/debops.nfs_server/getting-started.html#example-inventory), for example.